stable_baselines3/common/torch_layers.py

```diff
@@ -55,19 +55,31 @@ class NatureCNN(BaseFeaturesExtractor):
     :param observation_space:
     :param features_dim: Number of features extracted.
         This corresponds to the number of unit for the last layer.
+    :param normalized_image: Whether to assume that the image is already normalized
+        or not (this disables dtype and bounds checks): when True, it only checks that
+        the space is a Box and has 3 dimensions.
+        Otherwise, it checks that it has expected dtype (uint8) and bounds (values in [0, 255]).
```

Comment:
Do you think that we have to check that dtype is a Box when `normalized_image` is True?

Reply:
dtype is either `float32` os `uint8`, we check that the obs space is a box in all cases.

Reply:
typo:
Do you think that we have to check that dtype is a float32 when normalized_image is True?

Reply:
not sure, I think it doesn't matter as `obs.float()` is called in pre-processing afterward, and at that point the user should better know what he/she is doing (passing `normalize_images=False` is an active choice).